t/deprecation-warnings.t

```diff
@@ -7,38 +7,68 @@ run_tests();
 __DATA__
 
 === TEST 1: deprecation warnings
-APIcast should emit deprecation warnings when loading code using the old paths.
+APIcast should emit deprecation warnings when loading renamed files.
 --- configuration
-{
-  "services": [
-    {
-      "backend_version": 1,
-      "proxy": {
-        "policy_chain": [
-          { "name": "policy.echo" },
-          { "name": "apicast" }
-        ],
-        "proxy_rules": [
-          { "pattern": "/", "http_method": "GET", "metric_system_name": "hits", "delta": 2 }
-        ]
+{ "services": [
+  { "proxy":
+    {  "policy_chain": [
+      { "name": "apicast.policy.upstream",
+        "configuration": {
+          "rules": [{
+            "regex": "/",
+            "url": "http://test:$TEST_NGINX_SERVER_PORT"
+          }]
+        }
       }
-    }
-  ]
-}
---- backend
-  location /transactions/authrep.xml {
+    ] }
+  }
+] }
+--- upstream
+  location /{
     content_by_lua_block {
+      require('apicast')
       ngx.exit(200)
     }
   }
 --- request
 GET /echo?user_key=foo
---- response_body
-GET /echo?user_key=foo HTTP/1.1
 --- error_code: 200
 --- no_error_log
 [error]
 --- grep_error_log eval: qr/DEPRECATION:[^,]+/
 --- grep_error_log_out
-DEPRECATION: when loading apicast code use correct prefix: require("apicast.policy.echo")
 DEPRECATION: file renamed - change: require("apicast") to: require("apicast.policy.apicast")
+
+
+=== TEST 2: deprecation warnings
+APIcast should emit deprecation warnings when loading not namespaced code.
+--- configuration
+{ "services": [
+  { "proxy":
+    {  "policy_chain": [
+      { "name": "apicast.policy.upstream",
+        "configuration": {
+          "rules": [{
+            "regex": "/",
+            "url": "http://test:$TEST_NGINX_SERVER_PORT"
+          }]
+        }
+      }
+    ] }
+  }
+] }
+--- upstream
+  location /{
+    content_by_lua_block {
+      require('cli')
```

Comment:
I'm a bit confused bit this change. A policy chain with `{ "name": "policy.echo" }` like we had before will still make Apicast show a deprecation error, right?

Reply:
yes in this PR still, but in will need some touches in https://github.com/3scale/apicast/pull/566
This test is about prefixing it with `apicast`, so this example is more stable than the policies one.

Reply:
OK. Didn't think about the implications of this in the other PR.